cmd/kpod/ps.go

```diff
@@ -280,61 +391,86 @@ func getContainers(containers []*libkpod.ContainerData, opts psOptions) []*libkp
 }
 
 // getTemplateOutput returns the modified container information
-func getTemplateOutput(containers []*libkpod.ContainerData, opts psOptions) (psOutput []psTemplateParams) {
+func getTemplateOutput(containers []*libpod.Container, opts psOptions) ([]psTemplateParams, error) {
+	var psOutput []psTemplateParams
 	var status string
 	for _, ctr := range containers {
-		ctrID := ctr.ID
-		runningFor := units.HumanDuration(time.Since(ctr.State.Created))
+		ctrID := ctr.ID()
+		conConfig, err := ctr.GetConfig()
+		if err != nil {
+			return psOutput, errors.Wrapf(err, "unable to obtain container config")
+		}
+		conState, err := ctr.State()
+		if err != nil {
+			return psOutput, errors.Wrapf(err, "unable to container state")
```

Comment:
"unable to get container state" missing 'get'.

Reply:
fixed